code/datums/datacore.dm

```diff
@@ -349,6 +349,8 @@ GLOBAL_VAR_INIT(record_id_num, 1001)
 
 	preview_icon.Blend(face_s, ICON_OVERLAY)
 
+	var/icon/hands_icon = icon(preview_icon)
+	hands_icon.Blend(icon('icons/mob/clothing/masking_helpers.dmi', "l_hand_mask"), ICON_MULTIPLY)
```

Comment:
will this work properly with different species? like vox?

Reply:
Its the same case as with character preview in the lobby, even before the PR, so it won't

![image](https://user-images.githubusercontent.com/71735193/207983594-decb7667-449f-4a92-a064-9c355285a7d2.png)

![image](https://user-images.githubusercontent.com/71735193/207983724-3df8fa7d-4e9a-4035-8c24-6ba1eeaa60c8.png)


making it work with species variants would be an infinitely larger refactor, that, and I hope you won't blame me for that, I don't have time, energy, nor perhaps the skill to undertake, sorry.